docs/FAQ.md

```diff
@@ -108,6 +108,29 @@ You could refer to the above section to try out each of the below prefixes!
 
 **NOTE:** Currently, the above autocompletes ignore a text containing whitespaces if it is specified as a prefix for autocompletion. (e.g., `#test he` wouldn't lead to `#**test here**`) See also [#925](https://github.com/zulip/zulip-terminal/issues/925).
 
+#### Other autocomplete options
+
+All other message composing areas that support autocompletion don't require a prefix, i.e., the prefix for them is an empty string (`''`). Autocompletes here are mostly contextual and specific. See the below snapshots to better distinguish between various compose fields.
```

Comment:
Rather than prefixes being the empty string, we can just not mention prefixes in this case, or that they're taken from the context (you have "contextual" :+1:).